dNet/WorldPackets.h

```diff
@@ -10,6 +10,21 @@ struct SystemAddress;
 enum class eGameMasterLevel : uint8_t;
 enum class eCharacterCreationResponse : uint8_t;
 enum class eRenameResponse : uint8_t;
+namespace RakNet {
+	class BitStream;
+};
+
+#pragma pack(push, 1)
```

Comment:
Why?  The only reason you would use this is so you could write the struct as is to the stream, but you have a serialize anyways, as such this isnt necessary and can be removed.

Reply:
Iirc, you showed me this months ago cause it was cool lol

Reply:
While it is cool (and live accurate) let's go for the safer programmatic approach